source/speaker-diversity.rst

```diff
@@ -0,0 +1,69 @@
+Speaker diversity
+=====================================
+
+PyCon UK 2016 aims to present a high-quality and balanced programme of talks and other sessions
+(workshops, sprints, panels).
+
+These sessions will represent the diversity of interests within the Python community, and also the
```

Comment:
> will

I appreciate the bullishness, but given how easy it is to fuck up, perhaps "aim to"?